board/piksiv3/rootfs-overlay/usr/bin/dump_core.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+/usr/bin/logger "CORE DUMP!! executable=$1 pid=$2 time=$3"
+echo "CORE DUMP!! executable=$1 pid=$2 time=$3" | /usr/bin/sbp_log --error
+
+corefile="/tmp/cores/core-$1-$2-$3"
+touch $corefile
+
+while read line
+do
```

Comment:
Apart from limiting the size of the core file, the cores could be dumped on a size limited tmpfs:
```
mount -t tmpfs -o size=16m none /tmp/cores
```